etc/zulip.profile

```diff
@@ -0,0 +1,46 @@
+# Firejail profile for zulip
+# Description: Real-time team chat based on the email threading model
+# This file is overwritten after every install/update
+# Persistent local customizations
+include zulip.local
+# Persistent global definitions
+include globals.local
+
+ignore noexec /tmp
+
+noblacklist ${HOME}/.config/Zulip
```

Comment:
Add `blacklist ${HOME}/.config/Zulip` to disable-programs.inc (file is alphabetically sorted)

Reply:
Added; thank you, and sorry, it's been a while since I hacked on firejail :smile: